packages/twenty-front/src/modules/views/view-picker/components/ViewPickerListContent.tsx

```diff
@@ -56,29 +63,61 @@ export const ViewPickerListContent = () => {
 
   const { getIcon } = useIcons();
 
+  const handleDragEnd = useCallback(
+    (result: DropResult) => {
+      if (!result.destination) return;
+
+      moveArrayItem(viewsOnCurrentObject, {
+        fromIndex: result.source.index,
+        toIndex: result.destination.index,
+      }).forEach((view, index) => {
+        if (view.position !== index) {
+          updateView({ ...view, position: index });
+        }
+      });
+    },
+    [updateView, viewsOnCurrentObject],
+  );
+
+  const getSortedViews = () => {
```

Comment:
we should not recode this logic. It's already in: getObjectMetadataItemViews function